### ISSUE TITLE
UI: Can't open job/resource details from pipeline view

### ISSUE THREAD

**Issue by JohannesRudolph** [concourse/concourse#1785]
# Bug Report

Since upgrading to v3.6.0 I'm sometimes not able to open jobs/resources on the pipeline. I can see that the browser interprets the anchor tag on the dom element correctly and e.g. on Chrome I can ctrl+click it to open the right resource/job detail page in a new tab. 

* Concourse version: 3.6
* Deployment type: Docker
* Infrastructure/IaaS: Meshcloud OpenStack
* Browser (if applicable): Chrome
* Did this used to work? Yes, no problems in 3.5

**Comment by jtarchie** [comment]
@JohannesRudolph, are you able to recreate the issue on [ci.concourse.ci](https://ci.concourse.ci]? I'm using chrome on a Ubuntu machine and don't see the issue. Just trying to narrow down variables. Thanks.

**Comment by JohannesRudolph** [comment]
I just had a look, unfortunately can't reproduce it there. It also doesn't
happen consistently in our environment. Sometimes a page reload fixes it,
sometimes it doesn't....

What's really weird is that I just tried to narrow it down using chrome dev
tools while I was seeing the issue and it appears that opening the dev
tools made the issue go away? Maybe also be that something went bonkers
with chromes svg handling...

**Comment by JohannesRudolph** [comment]
Okay, so I found that the workaround is to trigger a "Inspect element" by right-clicking on it via the Chrome dev tools. I'm using Chrome 62.0.3202.75 on Windows 10. 

Looks like a browser bug to me to be fair.

**Comment by JohannesRudolph** [comment]
I just checked in with my coworkers that use Firefox as their primary browser, no issues reported by them.

**Comment by vito** [comment]
Are you sure you're not trying to do this from the `/beta` routes? Those aren't currently supported.

**Comment by vito** [comment]
Closing this as a repro seems unlikely at this point.

Maybe it's a faulty chrome extension?

**Comment by aeijdenberg** [comment]
I encountered this issue yesterday when kicking off a build from my home Mac. I had just run the High Sierra update the evening prior, and then when I wanted to click on a job in Concourse, nothing happening when I clicked on them. I tried page reloading - checked the JS console and saw nothing, even tried in Firefox (I was previously in Chrome 63.something). Restarting browser and/or reloading page didn't help. Nothing interesting I could see in the `atc` logs either.

Eventually I used my Chromebook laptop instead and everything was fine.

Today I'm back in the office, and on a Mac with High Sierra and Chrome 63.something and everything is fine.

Let me know if there's any other info you'd like me to collect when I'm back on the offending workstation. As far as I'm aware, I'm not running any Chrome extensions (and Firefox was also affected).
